tests/test_manager.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for manager."""
+from unittest import TestCase
+from ecosystem.entities import Repository
+from ecosystem.manager import Manager
+
+
+class TestManager(TestCase):
+    """Test class for manager functions."""
+
+    def test_issue_parsing(self):
+        """"Tests issue parsing function"""
+        issue_body = """### Github repo
```

Comment:
Let's move this unindented text in separate file and read it on setup of test.

Something like:
```python
class TestManager(TestCase):
    """Test class for manager functions."""

    def setUp(self) -> None:
         with open(<PATH_TO_FILE_WITH_ISSUE_BODY>, "r") as issue_body_file:
             self.issue_body = issue_body_file.read()

    def test_issue_parsing(self):
        """"Tests issue parsing function"""
        parsed_result = Manager.parse_submission_issue(self.issue_body)
```

Reply:
done